next.config.js

```diff
@@ -21,6 +21,16 @@ const nextConfig = {
           "@buf/cosmos_cosmos-sdk.bufbuild_es",
           "@buf/evmos_evmos.bufbuild_es",
           "@buf/cosmos_ibc.bufbuild_es",
+          "wagmi",
+          "@tanstack/query-sync-storage-persister",
+          "@tanstack/react-query",
+          "@tanstack/query-core",
+          "@tanstack/react-query-persist-client",
+          "@tanstack/query-persist-client-core",
+          "@wagmi/core",
+          "@wagmi/connectors",
+          "viem",
+          "abitype",
```

Comment:
This is all just due to weird CJS vs ESM interactions when running Jest. I think I'm going to add a task to migrate to vitest since it doesn't have these problems and I've had a good experience with it in the skip-router repo